packages/design-tokens/src/scss/generated/_radii.scss

```diff
@@ -3,10 +3,12 @@ $radius-0: 0 !default;
 $radius-100: 4px !default;
 $radius-200: 8px !default;
 $radius-300: 12px !default;
+$radius-50: 2px !default;
```

Comment:
@crishpeen Can we provide numeric sorting in these cases? 🤔 I can see [measures](https://github.com/lmc-eu/jobs-design-system/blob/main/packages/design-tokens/src/scss/generated/_measures.scss) are sorted correctly:

```scss
$space-0: 0 !default;
$space-100: 1px !default;
$space-200: 2px !default;
$space-300: 4px !default;
$space-400: 8px !default;
$space-500: 12px !default;
$space-600: 16px !default;
$space-700: 24px !default;
$space-800: 32px !default;
$space-900: 40px !default;
$space-1000: 48px !default;
$space-1100: 56px !default;
$space-1200: 64px !default;
$space-1300: 72px !default;
$space-1400: 80px !default;
```

Reply:
cc @literat

Reply:
We can fix this easily. It is the same as in measures, as @adamkudrna mentioned already. Give me a sec. ;)

Reply:
https://github.com/lmc-eu/spirit-design-system/pull/555

Reply:
Solved!

Reply:
@crishpeen I will enjoy your next billing :D